test/widgets/login_test.dart

```diff
@@ -113,6 +113,27 @@ void main() {
           id: testBinding.globalStore.accounts.single.id));
     });
 
+    testWidgets('account already exists', (tester) async {
+      final zulipLocalizations = lookupZulipLocalizations(ZulipLocalizations.supportedLocales.first);
```

Comment:
We have a somewhat more concise pattern we use for this:
```suggestion
      final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
```

From a grep, I see there's one existing place in the tests where we do the more verbose thing, in `test/api/core_test.dart`. From `git log -p -S Localizations test/` I see that's the very first time we wrote a test that compared to a localized string. A separate small, [NFC](https://github.com/zulip/zulip-mobile/blob/main/docs/glossary.md#nfc), commit to just update that one spot would be welcome to add to this PR.